.github/workflows/test.yml

```diff
@@ -40,43 +40,53 @@ jobs:
       - name: Install dependencies
         run: yarn install --frozen-lockfile --ignore-scripts
       - name: Check Formatting
-        run: yarn prettier -c pages src
+        run: yarn prettier -c .
 
-  test:
+  types:
     runs-on: ubuntu-latest
     needs: lint
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - uses: dorny/paths-filter@v2
-        id: changes
+      - name: Install Node.js
+        uses: actions/setup-node@v3
         with:
-          filters: |
-            server:
-              - 'src/server/**'
-            scripts:
-              - 'src/scripts/**'
-            assets:
-              - 'assets/**'
+          node-version: 16
+          cache: yarn
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile --ignore-scripts
```

Comment:
needs to explicitely run svgo script to typecheck generated components. Once the frontend uses TS this jobs fails with the components existing!